travel_journey/report/travel_journey_webkit.py

```diff
@@ -40,20 +40,37 @@ def _get_signer(self):
         except (KeyError, IndexError):
             return ''
 
+    # Use a class method: method that is not bound
+    # to an object,bbut to… a class!
+    @classmethod
```

Comment:
If you use a class method, `self` should be `cls`
See https://docs.python.org/2/library/functions.html#classmethod